std-bits/base/src/main/java/org/enso/base/Time_Utils.java

```diff
@@ -68,14 +62,46 @@ public static LocalDate date_adjust(LocalDate date, long add, Period duration) {
     return add == 1 ? date.plus(duration) : date.minus(duration);
   }
 
-  public static long week_of_year(LocalDate date, Locale locale) {
+  public static ZonedDateTime datetime_adjust(
+      ZonedDateTime datetime, long add, Period period, Duration duration) {
+    return add == 1 ? datetime.plus(period).plus(duration) : datetime.minus(period).minus(duration);
+  }
```

Comment:
I'm a bit hesitant of the usage of `long add` like this. If I set `add` to `0` or `100` it will have the same effect as `-1` - not very obvious.

Maybe use a `boolean forward` flag instead? Or ideally maybe we can just create an `enum Direction` for this?

Reply:
I followed the pattern used in Date by @JaroslavTulach but sure, I can tweak it a bit